NTUEEInfoDep/NTUEECourseMap
# [Backend] group mutation & init data

### What is this PR for?

1. add mutation resolver of group
2. init db with data
### What type of PR is it?
[Feature]

### Todos

### What is the GitHub issue?

#46 
### How should this be tested?


After opening apollo graphql playground, type `pnpm initData` to init db and use resolver `AllGroup` to check whether the data is created successfully

### Screenshots (if appropriate)

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? Yes
